Source/WebCore/dom/make_names.pl

```diff
@@ -1048,6 +1048,8 @@ sub printElementNameHeaderFile
     print F "} // namespace ElementNames\n";
     print F "\n";
     print F "ElementName findElementName(Namespace, const String&);\n";
+    print F "ElementName findHTMLElementNameFromBuffer(Span<const LChar>);\n";
+    print F "ElementName findHTMLElementNameFromBuffer(Span<const UChar>);\n";
```

Comment:
Can these just be `findHTMLElementName`?  I don’t think we need “from buffer” in the name.